src/overview/sidebar/components/ButtonContainer.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import styles from './ButtonContainer.css'
+
+class SidebarIcons extends PureComponent {
+    render() {
+        return (
+            <div
+                className={cx(styles.buttonContainerOverview, {
+                    [styles.buttonContainer]: !this.props.overviewMode,
+                })}
+            >
+                <div
+                    style={{
```

Comment:
This code looks great, although I would move these inline styles into the CSS module (unless there's a reason they're different from the others?). And I don't see why the module's name is `ButtonContainer.jsx` (the corresponding CSS module is also named the asme), although the component's name is `SidebarIcons`. Maybe just rename the module to `SidebarIcons.jsx`

Reply:
Ah one more (very minor) thing: as this is a `class`, might as well put your `propTypes` as a `static` prop to be consistent with all the other class components.

Reply:
There is a different component `SidebarIcon` which is for icons that will be shown in the overview, and the `ButtonContainer` basically contains for the buttons to change the state of the sidebar, so I can name the class `ButtonContainer`, I copied this from the `SidebarIcon` forgot to rename it.